Assets/HoloToolkit-Examples/GazeRuler/Scripts/IClosable.cs

```diff
@@ -0,0 +1,9 @@
+﻿using UnityEngine;
+
+/// <summary>
+/// any class inherit this interface should be closeable
+/// </summary>
+public interface IClosable  {
```

Comment:
Did you mean to inherit from IClosable? Perhaps a better name may be? IPolygonClose or something?